<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LIN65004458" failures="6" tests="7" name="Submit order" time="160.997" errors="0" timestamp="2025-02-06T17:40:50 IST">
  <testcase classname="frameworks.ActualE2ETest" name="submitOrderTest" time="23.983">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at frameworks.ActualE2ETest.submitOrderTest(ActualE2ETest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- submitOrderTest -->
  <testcase classname="frameworks.ActualE2ETest" name="submitOrderTest" time="23.868">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at frameworks.ActualE2ETest.submitOrderTest(ActualE2ETest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- submitOrderTest -->
  <testcase classname="frameworks.ActualE2ETest" name="submitOrderHashMapTest" time="23.627">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at frameworks.ActualE2ETest.submitOrderHashMapTest(ActualE2ETest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- submitOrderHashMapTest -->
  <testcase classname="frameworks.ActualE2ETest" name="submitOrderHashMapTest" time="23.646">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at frameworks.ActualE2ETest.submitOrderHashMapTest(ActualE2ETest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- submitOrderHashMapTest -->
  <testcase classname="frameworks.ActualE2ETest" name="submitOrderJSONTest" time="23.694">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at frameworks.ActualE2ETest.submitOrderJSONTest(ActualE2ETest.java:83)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- submitOrderJSONTest -->
  <testcase classname="frameworks.ActualE2ETest" name="submitOrderJSONTest" time="12.661">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at frameworks.ActualE2ETest.submitOrderJSONTest(ActualE2ETest.java:83)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- submitOrderJSONTest -->
  <testcase classname="frameworks.ActualE2ETest" name="orderHistoryTest" time="0.0">
    <skipped/>
  </testcase> <!-- orderHistoryTest -->
</testsuite> <!-- Submit order -->
